### DISCUSSION TITLE
Is it possible to mix Skia-canvas with FabricJS?

### DISCUSSION THREAD

**Discussion by ProgrammingLife** [samizdatco/skia-canvas#71] [Q&A]
I wanna use FabricJS on top of Skia-canvas for SSR. 
FabricJS initializes its canvas with the following way:
```
const canvas = new fabric.Canvas('c');
canvas.backgroundColor = 'yellow';
```
I don't undestand how to properly pass context from Skia to FabricJS. Any ideas?

**Comment by Mat-thieu** [comment]
It's an old thread, but if anyone stumbles upon it the canvas registration and detection is actually done by JSdom, a dependency used by Fabric.
It assumes `"canvas"` in `JSdom/lib/jsdom/utils.js`. I'd suggest you fork JSdom and update this import to target `skia-canvas` instead, then you need to fork `fabric.js` and update the JSdom dependency with your own fork.
